analysis.R

```diff
@@ -1,138 +1,129 @@
-###################
-##  12/27/21  ##
-###################
+#####################################
+##  Depression-Belief Interaction  ##
+#####################################
 
 ####  Startup  ####
-
 # Load packages
-
-if(!require(tidyverse)){install.packages('tidyverse')}
-if(!require(here)){install.packages('here')}
-if(!require(ggplot2)){install.packages('ggplot2')}
-
 library(tidyverse)
 library(here)
 library(ggplot2)
 
 
-#### Load data ####
-
-# The here() package returns filepaths based on the arguments you give it,
-# starting from the folder with your .Rproj file; in this case, it will return
-# the filepath to the dataset inside the "data" folder
-# This is especially useful when a script will be run on Windows and Mac 
-# computers, as they use different folder separators ("/" vs "\")
 
+#### Load data ####
 df_1 <- readRDS(here("data", "T2T Data for Jans et al.rds"))
 
 
-####  Prepare Data  ####
 
+####  Prepare Data  ####
+df_2 <- df_1 %>%
+  
   # Adding a column for CDI mean
-
-  df_2 <- df_1 %>% mutate(cdi_mean = rowMeans(select(.,contains("cdi")), na.rm = FALSE))
+  mutate(cdi_mean = rowMeans(select(.,contains("cdi")), na.rm = FALSE)) %>%
```

Comment:
All of these steps can actually be done in one big set of functions called a "pipe" using the `%>%` operator